src/components/clusterWizard/ClusterWizardForm.tsx

```diff
@@ -89,63 +103,131 @@ const ClusterWizardForm: React.FC<ClusterWizardFormProps> = ({ cluster, setStep
+                      helperText="Virtual IP used internally by the cluster for automating internal DNS requirements."
+                      isRequired
+                    />
+                    <InputField
+                      name="ingressVip"
+                      label="Ingress Virtual IP"
+                      helperText="Virtual IP used for cluster ingress traffic."
                       isRequired
                     />
-                    <InputField name="dnsVIP" label="Internal DNS Virtual IP" isRequired />
-                    <InputField name="ingressVIP" label="Ingress Virtual IP" isRequired />
-                    <InputField name="machineCIDR" label="Machine CIDR" placeholder="" isRequired />
                     <TextContent>
                       <Text component="h2">Security</Text>
                     </TextContent>
-                    <TextAreaField name="pullSecret" label="Pull Secret" isRequired />
-                    <TextAreaField name="sshPrivateKey" label="SSH Private Key" isRequired />
-                    <TextAreaField name="sshPublicKey" label="SSH Public Key" isRequired />
+                    <TextAreaField
+                      name="pullSecret"
+                      label="Pull Secret"
+                      helperText={
+                        <>
+                          The pull secret obtained from the Pull Secret page on the{' '}
+                          {
+                            <a
+                              href="https://cloud.redhat.com/openshift/install/metal/user-provisioned"
+                              target="_blank"
+                              rel="noopener noreferrer"
+                            >
+                              Red Hat OpenShift Cluster Manager site <ExternalLinkAltIcon />
+                            </a>
+                          }
+                          .
+                        </>
+                      }
+                      isRequired
+                    />
+                    <TextAreaField
+                      name="sshPublicKey"
+                      label="SSH Public Key"
+                      helperText="SSH public key for debugging OpenShift nodes."
+                      isRequired
+                    />
                   </GridGap>
                 </GridItem>
               </Grid>
             </Form>
           </PageSection>
+          <AlertsSection>
+            {status.error && (
+              <Alert
+                variant={AlertVariant.danger}
+                title={status.error}
+                action={<AlertActionCloseButton onClose={() => setStatus({ error: null })} />}
+              />
+            )}
+          </AlertsSection>
           <ClusterWizardToolbar>
             <ToolbarButton
               variant={ButtonVariant.secondary}
-              onClick={() => setStep(WizardStep.BaremetalInventory)}
+              component={(props) => <Link to="/clusters" {...props} />}
```

Comment:
What about referring the URL via a constant? We are still discussing the future deployment of the app and this might be affected.

Reply:
The transition in the wizard is currently done via state not router, the `/cluster` route points to `ClusterPage` component.

At some point we can decide to drive it via routes but I kept it in state for now so we're in control of how we allow the user to transition between the steps.

Reply:
I agree we should turn the routes into constants (`config/routes` maybe?) probably in separate PR?